substrate/.maintain/frame-weight-template.hbs

```diff
@@ -33,7 +33,7 @@ pub trait WeightInfo {
 
 /// Weights for `{{pallet}}` using the Substrate node and recommended hardware.
 pub struct SubstrateWeight<T>(PhantomData<T>);
-{{#if (eq pallet "frame_system")}}
+{{#if (or (eq pallet "frame_system") (eq pallet "frame_system_extensions"))}}
 impl<T: crate::Config> WeightInfo for SubstrateWeight<T> {
```

Comment:
I have not looked closely enough at everything yet, but the vibes I am getting is that the whole story of "how to benchmark tx extensions" is not very solid. 

For example, what if a pallet other than system declares a tx-extenion?

Reply:
This is the "special handling" of `frame_system` extensions, just like `frame_system` is handled differently because the pallet name isn't "pallet-*`, it's "frame_system". These things matter because there is a script looking up names and pasting the weight functions and they have to match.

If you look at other extensions, like `ChargeTransactionPayment` or `CheckOnlySudo`, they are benchmarked within their own pallet and nothing extra needs to be done. However, there will be no way to get around the fact that pallet-less extensions are a pain to benchmark, because users will have to create a pallet that they include in their runtime just for benchmarking purposes.